activities/resolution_hierarchy.py

```diff
@@ -12,147 +12,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import blosc
-import hashlib
-import boto3
-import botocore
-import numpy as np
-from PIL import Image
-from scipy.ndimage.interpolation import zoom
-from collections import namedtuple
 from bossutils import aws, logger
-from spdb.c_lib import ndlib
 from spdb.c_lib.ndtype import CUBOIDSIZE
 
-from multidimensional import XYZ, Buffer, ceildiv
-from multidimensional import range as xyz_range
+from bossutils.multidimensional import XYZ, ceildiv
+from bossutils.multidimensional import range as xyz_range
 
-log = logger.BossLogger().logger
+from heaviside.activities import fanout
 
-np_types = {
-    'uint64': np.uint64,
-    'uint16': np.uint16,
-    'uint8': np.uint8,
-}
+log = logger.BossLogger().logger
 
-# BOSS Key creation functions / classes
-def HashedKey(*args, version = None):
-    """
-    Args:
-        collection_id
-        experiment_id
-        channel_id
-        resolution
-        time_sample
-        morton (str): Morton ID of cube
+POLL_DELAY = 5
```

Comment:
Can you add comments or add to the constants' name the unit of time?

Reply:
Added comments